tree/ntuple/src/RNTupleProcessor.cxx

```diff
@@ -336,6 +336,21 @@ void ROOT::Experimental::RNTupleChainProcessor::AddEntriesToJoinTable(Internal::
 
 //------------------------------------------------------------------------------
 
+namespace ROOT::Experimental::Internal {
+class RAuxiliaryProcessorField final : public ROOT::RRecordField {
```

Comment:
As far as I understand, this field is only meant for reading. You can add
```c++
   void GenerateColumns() final { throw RException(R__FAIL("RAuxiliaryProcessorField must only be used for reading")); }
```
as an additional safeguard.

Reply:
I had this additionally, but the debian build didn't like it because of it being a hidden virtual overloaded function. The now-added `using` statement solves this though!